native/packages/react-native-bpk-component-card/index.js

```diff
@@ -15,7 +15,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
```

Comment:
it's be good if we could add a rule to eslint to allow `.android.js` & `.ios.js`

Reply:
I'm trying this https://github.com/benmosher/eslint-plugin-import/issues/279#issuecomment-215052176
happy for me to add this to the `.eslintrc`?